server/app/services/export/JsonExporter.java

```diff
@@ -119,106 +126,49 @@ private CfJsonDocumentContext buildJsonApplication(
             () -> jsonApplication.putNull(statusPath));
 
     for (AnswerData answerData : roApplicantProgramService.getSummaryData()) {
-      // Answers to enumerator questions should not be included because the path is incompatible
-      // with the JSON export schema. This is because enumerators store an identifier value for
-      // each repeated entity, which with the current export logic conflicts with the answers
-      // stored for repeated entities.
-      if (answerData.questionDefinition().isEnumerator()) {
-        continue;
+      exportToJsonApplication(jsonApplication, answerData);
+    }
+
+    return jsonApplication;
+  }
+
+  private void exportToJsonApplication(
+      CfJsonDocumentContext jsonApplication, AnswerData answerData) {
+    @SuppressWarnings("unchecked")
+    ImmutableMap<Path, ?> entries =
```

Comment:
**nitpick (blocking):** please add a comment explaining why the unchecked warning needs to be suppressed

Reply:
Done.